themes/src/main/resources/theme/base/login/resources/js/utils.js

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Do we really need to add this copyright notice? This code will get shipped to the users' browser, so best to keep it as minimal as possible for performance reasons.